packages/devtools_app/lib/src/screens/inspector/inspector_screen.dart

```diff
@@ -565,6 +569,63 @@ class _ErrorNavigatorButton extends StatelessWidget {
   }
 }
 
+class InspectorDetailsOption extends StatelessWidget {
```

Comment:
Could we rename to something more descriptive like "DefaultTabOption"?

Reply:
Ahh I think just renaming to InspectorDetailsViewOption would be more clear (then everything else wouldn't have to be renamed)